packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -349,21 +449,38 @@ export const getActivitiesByAddress = async function (
               (e) => e.token.type.toLowerCase() === TokenERC.ERC20
             );
             const erc20TxTotal = erc20Tx?.total as BlockscoutTotalERC20;
+            const decimals = erc20Tx?.token.decimals
+              ? Number(erc20Tx?.token.decimals)
+              : 0;
```

Comment:
We shouldn't assume decimals to be `0`. In case of missing information, it's better to skip including the fiatValue.